apps/condo/domains/ticket/queries/IncidentProperty.graphql

```diff
@@ -1,8 +1,62 @@
-query getIncidentProperties ($where: IncidentPropertyWhereInput) {
-    incidentProperties: allIncidentProperties (where: $where) {
+query getIncidentPropertiesByIncidentId ($incidentId: ID!) {
+    incidentProperties: allIncidentProperties (where: {
+        incident: {
+            id: $incidentId,
+        }
+    }) {
         id
         incident {
             id
         }
+        property {
+            id
+            addressMeta {
+                value
+            }
+            address
+        }
+        propertyAddress
+        propertyAddressMeta {
+            value
+        }
+    }
+}
+
+query getIncidentPropertiesByPropertyIdAndRecentIncidents ($propertyId: ID!, $organizationId: ID!, $workFinish_gte: String) {
```

Comment:
mb use <ins>incident</ins>WorkFinish_gte? 
Like <ins>property</ins>Id, <ins>organization</ins>Id

What do you think?